hastory-server/src/Data/Hastory/Server/Handler/Import.hs

```diff
@@ -2,13 +2,15 @@ module Data.Hastory.Server.Handler.Import
   ( module X
   ) where
 
-import Control.Monad.Except as X
-import Database.Persist as X
-import Servant.API as X hiding (BasicAuth)
-import Servant.Auth.Server as X
-import Servant.Server as X hiding (BasicAuthResult(..))
+import           Control.Monad.Except      as X
+import           Database.Persist          as X
+import           Servant.API               as X hiding (BasicAuth)
+import           Servant.Auth.Server       as X
```

Comment:
This is a bit silly, be sure to set up the pre-commit hook using `nix-shell`. Then this doen't end up in the commit.

Reply:
Sorry - I usually run the pre-commit hook. There was an extra commit (f1eeafbb) with only formatting changes.